cmd/prometheus/main.go

```diff
@@ -415,6 +415,9 @@ func main() {
 	serverOnlyFlag(a, "storage.tsdb.retention.size", "Maximum number of bytes that can be stored for blocks. A unit is required, supported units: B, KB, MB, GB, TB, PB, EB. Ex: \"512MB\". Based on powers-of-2, so 1KB is 1024B.").
 		BytesVar(&cfg.tsdb.MaxBytes)
 
+	serverOnlyFlag(a, "storage.tsdb.retention.percentage", "Maximum percentage of the disk space that can be used to store blocks (prevails over storage.tsdb.retention.size).").
```

Comment:
```suggestion
	serverOnlyFlag(a, "storage.tsdb.retention.percentage", "Maximum percentage of disk used for TSDB storage (overrides storage.tsdb.retention.size).").
```

Then the more detailed commentary can go in `docs/storage.md`